src/multicellular_stage/editor/CellBodyPlanEditorComponent.cs

```diff
@@ -620,13 +676,44 @@ protected override void Dispose(bool disposing)
         base.Dispose(disposing);
     }
 
-    private void UpdateGUIAfterLoadingSpecies()
+    private void SetLightLevelOption(int option)
     {
-        GD.Print("Starting multicellular editor with: ", editedMicrobeCells.Count,
-            " cells in the microbe");
+        // Show selected light level
+        switch ((LightLevelOption)option)
+        {
+            case LightLevelOption.Day:
+            {
+                Editor.DayLightFraction = 1;
+                break;
+            }
 
-        SetSpeciesInfo(newName,
-            behaviourEditor.Behaviour ?? throw new Exception("Editor doesn't have Behaviour setup"));
+            case LightLevelOption.Night:
+            {
+                Editor.DayLightFraction = 0;
+                break;
+            }
+
+            case LightLevelOption.Average:
+            {
+                Editor.DayLightFraction = Editor.CurrentGame.GameWorld.LightCycle.AverageSunlight;
+                break;
+            }
+
+            case LightLevelOption.Current:
+            {
+                Editor.DayLightFraction = Editor.CurrentGame.GameWorld.LightCycle.DayLightFraction;
+                break;
+            }
+
+            default:
+                throw new Exception("Invalid light level option");
+        }
+    }
+
+    private bool IsNegativeAtpProduction()
+    {
+        return energyBalanceInfo != null &&
```

Comment:
Here, it doesn't help if there's one cell with massively positive ATP, because if there's any negative cell in the colony, that will just die when it grows in the body plan...